packages/loader/src/gltf/parser/GLTFAnimationParser.ts

```
@@ -41,15 +42,15 @@ export class GLTFAnimationParser extends GLTFParser {
     const entities = context.get<Entity>(GLTFParserType.Entity);
 
     let duration = -1;
-    let promises = new Array<Promise<void | Entity[]>>();
+    let promises = new Array<AssetPromise<void | Entity[]>>();
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)